core/domain/story_domain.py

```diff
@@ -59,6 +62,8 @@ class StoryChange(object):
         STORY_NODE_PROPERTY_PREREQUISITE_SKILL_IDS, STORY_NODE_PROPERTY_OUTLINE,
         STORY_NODE_PROPERTY_EXPLORATION_ID)
 
+    STORY_CONTENTS_PROPERTIES = (STORY_CONTENTS_PROPERTY_STARTING_NODE_ID)
```

Comment:
You will need a comma just before the last parenthesis if you want this to be treated as a tuple (and not a string).

https://wiki.python.org/moin/TupleSyntax

It is a surprise that this wasn't caught in tests. Before fixing this please write a test that contains a change that involves an element in this tuple and ensure that the test suite fails.

Reply:
There is already a test in test_update_story_node_properties() in story_services_test that changes the starting node id and it passes.

Reply:
That's weird. Why does the check here not trigger, then? https://github.com/oppia/oppia/pull/4988/files/4b22a75f039bc2a67ed214960c10ec30ab0e2a64..b79cee9522db6977e4a619f4e8c154fd125ae12b#diff-59604d490e874701fdc81e848603f084R111

EDIT: oh, I see. It's because you're comparing a string against the whole string. If you tried to use an invalid substring like 'TING_NODE' as the change cmd, I think it will go through.

Reply:
Done, made it a tuple.